Lesson03_Task01.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
Even without new line warning:
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py
./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py:1:1: D205 1 blank line required between summary line and description
./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py:7:80: E501 line too long (120 > 79 characters)
./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py:8:80: E501 line too long (94 > 79 characters)
./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py:17:80: E501 line too long (91 > 79 characters)
./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py:59:40: C812 missing trailing comma
./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py:73:7: P101 format string does contain unindexed parameters
./T-HOMETASKS/L03/l03_h01_MaximFedoseev.py:74:7: P101 format string does contain unindexed parameters